### DISCUSSION TITLE
SEPP docker-based deployement for 5G roaming

### DISCUSSION THREAD

**Discussion by MerouaM** [herlesupreeth/docker_open5gs#351] [General]
Hi everyone,

I’m curious if there are any plans to incorporate the SEPP NF (deployed on docker) to enable 5G roaming support. I’m keen on using this feature for my experimental work.

Thanks,

**Comment by herlesupreeth** [comment]
I plan to add it eventually but currently want to finish support for VoNR calling first and then onto other things. If you in a rush you can take a look at this PR - https://github.com/herlesupreeth/docker_open5gs/pull/332

**Reply by MerouaM** [reply]
Yes ! I am in a bit of a rush :). Thank you for the PR, I will have a look on it :)

**Reply by MerouaM** [reply]
I tried to follow the instructions in the PR, however, the sepp container is exiting with error code (255), and in the logs it shows:

`Deploying component: 'sepp-1'
08/07 11:28:47.606: [app] FATAL: Failed to parse configuration file '/open5gs/install/etc/open5gs/sepp.yaml' (../lib/app/ogs-init.c:188)
08/07 11:28:47.606: [app] ERROR: Scanner error - mapping values are not allowed in this context at line 5, column 7 (../lib/app/ogs-init.c:210)
08/07 11:28:47.606: [app] FATAL: Open5GS initialization failed. Aborted (../src/main.c:210)
`
Any idea on how to solve this?

@JulienSrcdImta @herlesupreeth
